### PR TITLE
Ported IPC wings from SimpleStation14

### PR DESCRIPTION
**Pull request by ScyronX**
## About the PR

Added wing marking for IPCs

**Changelog**

🆑 
- add: Added IPC wings

### CODE REVIEW

**Comment by deltanedas** [comment]
i wonder which character this is meant to be for

**Comment by HawkeyeBlade** [comment]
> i wonder which character this is meant to be for

IPCs, no?

File: Resources/Locale/en-US/markings/wings.ftl
Comment (Aidenkrz) [inline Resources/Locale/en-US/markings/wings.ftl:1]: this needs to be in goob namespace..

**Review by Aidenkrz — CHANGES_REQUESTED** [review]

**Comment by deltanedas** [comment]
> > i wonder which character this is meant to be for
> 
> IPCs, no? Is the code saying otherwise? XD

V1 ultrakill

File: Resources/Locale/en-US/markings/wings.ftl
Comment (ScyronX) [reply Resources/Locale/en-US/markings/wings.ftl:1]: Done

**Pull request merged**